DemoApp/CustomMessageContentView.swift

```diff
@@ -10,6 +10,11 @@ class CustomMessageContentView: ChatMessageContentView {
     override open func updateContent() {
         super.updateContent()
         
+        if content?.isShadowed == true {
+            textView?.textColor = appearance.colorPalette.textLowEmphasis
+            textView?.text = "This message is from a shadow banned user"
```

Comment:
nit: would be nice to get this localized 👍

Reply:
It's DemoApp only